examples/webgl_lightprobe.html

```diff
@@ -125,8 +126,8 @@
 
 					scene.background = cubeTexture;
 
-					lightProbe.setFromCubeTexture( cubeTexture );
-
+					lightProbe.copy( THREE.LightProbeGenerator.fromCubeTexture( cubeTexture ) );
```

Comment:
probably should be
```js
lightProbe = THREE.LightProbeGenerator.fromCubeTexture( cubeTexture );
```
but that requires a refactoring of the example.

Reply:
Yeah. Seemed like the simplest solution and it was not too bad 😇